frontend/glideinFrontendInterface.py

```diff
@@ -640,7 +639,7 @@ def get_key_obj(self, classad_identity, factory_pub_key_id, factory_pub_key, gli
         Returns:
 
         """
-        cache_id = factory_pub_key.get()
+        cache_id = factory_pub_key.string
```

Comment:
The cache_id should be the credential string or the id?

Reply:
This change was done to keep the exact same behavior of the `pubCrypto` module.